conf.py

```diff
@@ -65,7 +65,7 @@
 # The short X.Y version.
 version = '2.2'
 # The full version, including alpha/beta/rc tags.
-release = '2.2.0-beta.5'
+release = '2.2.0-beta.6'
```

Comment:
I think we can generally simply use `2.2.0` here instead of updating the documentation for every pre-release.